apps/core/src/hooks/useGetNextEpochCommitteeMember.ts

```diff
@@ -1,19 +1,23 @@
-// Copyright (c) 2025 IOTA Stiftung
-// SPDX-License-Identifier: Apache-2.0
-
 import { useIotaClientQuery } from '@iota/dapp-kit';
 import { useMemo } from 'react';
 import { useMaxCommitteeSize } from './useMaxCommitteeSize';
 
 export function useGetNextEpochCommitteeMember(validatorAddress: string): {
     isValidatorExpectedToBeInTheCommittee: boolean;
+    isValidatorExpectedToBeInTheCommitteeLoading: boolean;
```

Comment:
i would problably call this `isLoading` and then rename it wherever is used, there is no need for such a long name